changelog/20.0/20.0.0/summary.md

```diff
@@ -25,6 +25,7 @@
     - [Insert Row Alias Support](#insert-row-alias-support)
   - **[Query Timeout](#query-timeout)**
   - **[Flag changes](#flag-changes)**
+    - [`Pool Capacity Flags Deletion`](#pool-flags-deletion)
```

Comment:
We should probably move this to `Deletions` for more clarity and consistency with previous release notes

Reply:
Done.